vcr/holder/wallet_test.go

```diff
@@ -575,6 +575,50 @@ func Test_wallet_IsEmpty(t *testing.T) {
 	})
 }
 
+func Test_walletStore_remove(t *testing.T) {
+	engine := storage.NewTestStorageEngine(t)
+	require.NoError(t, engine.Start())
+	t.Run("ok", func(t *testing.T) {
+		resetStore(t, engine.GetSQLDatabase())
+		sut := New(nil, nil, nil, nil, storage.NewTestStorageEngine(t))
+
+		auditLogs := audit.CaptureLogs(t)
+
+		// Have 3 credentials in wallet, 2 of the subject wallet, 1 of another wallet
+		credentialToRemove := createCredential(vdr.TestMethodDIDA.String())
+		err := sut.Put(context.Background(), credentialToRemove)
+		require.NoError(t, err)
+		otherCredential1 := createCredential(vdr.TestMethodDIDA.String())
+		err = sut.Put(context.Background(), otherCredential1)
+		require.NoError(t, err)
+		otherCredential2 := createCredential(vdr.TestMethodDIDB.String())
+		err = sut.Put(context.Background(), otherCredential2)
+		require.NoError(t, err)
+
+		err = sut.Remove(audit.TestContext(), vdr.TestDIDA, *credentialToRemove.ID)
+		require.NoError(t, err)
+
+		// Make sure the other 2 credentials weren't removed
```

Comment:
should this also test the credential is not removed from the credentials table?

Reply:
There's a foreign key, so checked by the database constraints